identify_monographs.py

```diff
@@ -0,0 +1,402 @@
+        "Animated television programs",
+    ]
+    forms = _get_subfields(marc_record, "655", "a")
+    for form in forms:
+        if form in non_monograph_forms:
+            logger.debug(
+                f"Non-monograph form found for MMS ID {marc_record['001'].data}: "
+                f"{form}"
+            )
+            return True
+    # special case for 655 $a Anthologies and 655 $a Short films
+    if "Anthologies" in forms and "Short films" in forms:
+        return True
+    return False
+
+
+def _get_alma_monograph_forms(marc_record) -> list:
+    """Given a MARC record, returns a list of form values that are monograph types."""
+    monograph_forms = [
+        "Animation",
+        "Cartoons",
+        "Documentaries and factual films and video",
+        "Features",
+        "Feature films",
+        "Made for TV movies",
+        "Shorts",
+        "Specials",
+        "Animated films",
+        "Music shorts",
+        "Commercials",
+        "Television commercials",
+        "Theater advertising",
+    ]
+    # local monograph forms - only use these if $2 is "local"
+    local_monograph_forms = [
+        "Student films and video",
+    ]
+    forms = marc_record.get_fields("655")
+    monograph_form_values = []
+    for form in forms:
+        unpunctuated_a = form["a"].rstrip(".")
+        if form.get_subfields("2") == ["local"]:
+            if unpunctuated_a in local_monograph_forms:
+                monograph_form_values.append(form["a"])
+        elif unpunctuated_a in monograph_forms:
+            monograph_form_values.append(form["a"])
+    logger.debug(
+        f"Monograph forms for MMS ID {marc_record['001'].data}: {monograph_form_values}"
+    )
+    return monograph_form_values
+
+
+def _normalize_alma_monograph_form_values(form_values: list) -> list:
+    """Normalizes form values to a standard format."""
+    mapping = {
+        "Theater advertising": "Advertisements",
+        "Animated films": "Animation",
+        "Cartoons": "Animation",
+        "Commercials": "Television commercials",
+        "Documentaries and factual films and video": "Documentary",
+        "Features": "Feature films",
+        "Student films and video": "Student films",
+    }
+    normalized_form_values = []
+    for value in form_values:
+        # first, remove ending period, if present
+        value = value.rstrip(".")
+        if value in mapping:
+            normalized_form_values.append(mapping[value])
+        else:
+            normalized_form_values.append(value)
+    return normalized_form_values
+
+
+def _has_non_monograph_alma_title(marc_record) -> bool:
+    """Given a MARC record, checks whether the record has a title value indicating
+    a non-monograph type."""
+    # if $p or $n is in the title, it's not a monograph
+    marc_245 = marc_record.get_fields("245")
+    for field in marc_245:
+        if field["p"] or field["n"]:
+            return True
+    # if any 250 fields are present, it's not a monograph
```

Comment:
This could be simplified: `get_fields(*args)` returns a list of all fields matching any of `*args`, so just checking `get_fields("250", "505")` is enough.